src/main/ansible/vagrant.yml

```diff
@@ -60,14 +60,22 @@
       path: "{{ stores_dir }}/pdbs/ab"
     register: bag_dir_stat
 
-  - name: Copy test-bags to server
-    copy: # for now we have just one test bag, otherwise copy pdbs/bags into pdbs/ab and rename it
+  - name: Copy space-travel bag to server
+    copy: # bag-store knows it with uuid 40594b6d-8378-4260-b96b-13b57beadf7c
       src: "../../test/resources/vault/stores/pdbs/bags/9da0541a-d2c8-432e-8129-979a9830b427"
       dest: "{{ stores_dir }}/pdbs/ab/123456789012345678901234567890"
       mode: "0775"
       directory_mode: "0775"
     when: "bag_dir_stat.stat.exists == False"
 
+  - name: Copy test-bag to server
+    copy: # bag-store knows it with uuid ab123456-7890-1234-5678-901234567890
```

Comment:
Looks like the comment is swapped with the one above where the dest dir starts with 'ab'

Reply:
No, it is just a coincidence. I once thought the folder names (after `ab`/ respective `40/`) where the UUID's of the bags. They appeared to be unrelated. I discovered the actual values after indexing the bags in solar. As a convenience for manual tests I added the actual UUID's of the bags as comment.